### PR TITLE
feat(utils): adds the complete action type

### PR DESCRIPTION
**Pull request by jonathandiep**
Adds the `Complete` Action Type

### CODE REVIEW

File: packages/utils/src/types/actions.ts [hunk 1]
```diff
@@ -292,6 +294,26 @@ export const CreateActionDetailSchema = z.object({
 export type CreateActionForm = z.infer<typeof CreateActionFormSchema>
 export type CreateActionDetail = z.infer<typeof CreateActionDetailSchema>
 
+/*
+COMPLETE
+*/
+export type CompleteActionParams = {
+  boostId?: string
```
Comment (mmackz) [inline packages/utils/src/types/actions.ts:301]: I think we will also want to take in the chainId as an actionParam.

Comment (mmackz) [reply packages/utils/src/types/actions.ts:301]: Im not sure if we can make it work for all networks at the same time?

File: packages/utils/src/types/actions.ts [hunk 2]
```diff
@@ -292,6 +294,26 @@ export const CreateActionDetailSchema = z.object({
...
+  actionType?: string
+}
+
+export const CompleteActionFormSchema = z.object({})
```
Comment (mmackz) [inline packages/utils/src/types/actions.ts:305]: The form schema is for the inputs we want to take in on manager, (at least this is how I understand it)

So we are going to want to have optional fields here for boostId and actionType

File: packages/utils/src/types/actions.ts [hunk 3]
```diff
@@ -292,6 +294,26 @@ export const CreateActionDetailSchema = z.object({
...
+export const CompleteActionDetailSchema = z
+  .object({
+    boostId: z.string().optional(),
+    actionType: z.string().optional(),
+  })
+  .refine((data) => data.boostId != null || data.actionType != null, {
```
Comment (mmackz) [inline packages/utils/src/types/actions.ts:311]: We want the `chainId` in here also (assuming we want to add it to actionParams)

File: packages/utils/src/types/actions.ts (hunk 1 above)
Comment (jonathandiep) [reply packages/utils/src/types/actions.ts:302]: I added `chainId` as optional. It should allow us to enable for all networks. Will update if we need to enforce a `chainId`

**Review by mmackz — APPROVED** [review]
Looks good 🚢

**Pull request merged**